lessons/03-html-lists/index.html

```diff
@@ -0,0 +1,96 @@
+<html>
+<head>
+<title>Hogwarts</title>
+</head>
+
+<body>
+<h1>Hogwarts</h1>
+<h2>Houses</h2>
+
+<ul>
+<li><strong>Gryffindor</strong></li>
+<li>Hufflepuff</li>
+<li>Ravenclaw</li>
+<li>Slytherin</li>
+</ul>
+
+<h2>Notable teachers and staff members</h2>
+
+<ol>
+<li>Argus Filch</li>
+<li>Filius Flitwick</li>
+<li>Gilderoy Lockhart</li>
+<li>Minerva McGonagall</li>
+<li>Poppy Pomfrey</li>
+<li>Quirinus Quirrell</li>
+<li>Horace Slughorn</li>
+<li>Pomona Sprout</li>
+<li>Sybill Trelawney</li>
+</ol>
+
+
+<ul>
+  <li>Hogwarts ghosts
+     <ol>
+      <li>The Bloody Baron</li>
+      <li>The Fat Friar</li> 
+      <li>The Grey Lady</li> 
+      <li>Nearly Headless Nick</li>
+   </ol>
+  </li> 
+</ul>
+
+
+<ul>
+    <li>Hogwarts founders
+        <ol>
+        <li>Godric Gryffindor</li>
+        <li>Helga Hufflepuff</li> 
+        <li>Rowena Ravenclaw</li> 
+        <li>Salazar Slytherin</li>
+     </ol>
+    </li> 
+  </ul>
+  
+<h2>List of spells</h2>
+
+<dl>
```

Comment:
It appears this tag is missing it's closing half (`</dl>`) at the bottom of the file

Reply:
@nehero
If I change the code, then do I need to commit and push/pull again?